resources/styles/book-content/hs-physics/virtual-physics.less

```diff
@@ -1,48 +1,2 @@
- .virtual-physics {
-  background: @tutor-neutral-lightest;
-  margin: 68px 0 32px 0;
-  clear: both;
-  display: inline-block;
-  border-top: solid 8px @tutor-neutral-lighter;
-  border-bottom: solid 8px @tutor-neutral-lighter;
-  padding: 20px 40px;
-
-  &:before {
-    position: absolute;
-    content: attr(data-label);
-    #fonts > .sans(@reading-ui-banner-height - 18px, @reading-ui-banner-height);
-    font-weight: 900;
-    color:  @tutor-book-hs-physics-secondary;
-    background: @tutor-book-hs-physics-primary;
-    padding: 0 40px;
-    height: @reading-ui-banner-height;
-    display: inline-block;
-    margin: -68px 0 0 -40px;
-  }
-
-  :last-child {
-    padding-bottom: 0;
-  }
-
-  .title {
-    #fonts > .sans(3rem, 4.6rem);
-    font-weight: 900;
-  }
-
-  p,
-  ul,
-  li {
-    #fonts > .sans(1.8rem, 1.75em);
-    margin: 0 0  1.5rem 0;
-    color: @tutor-neutral-dark;
-  }
-  li {
-    padding: 5px 0;
-  }
-  .solution, .commentary {
-    [data-type=title]{
-      #fonts > .sans(1.8rem, 1.75em);
-      font-weight: bold;
-    }
-  }
+.virtual-physics {
```

Comment:
This has the empty class for the note (similarly to all the notes). Currently, none of them are "doing" anything but we will need them individually when we style each note separately. I can remove all the unused notes, but we will need to recreate them on a per needed basis. I initially created all the bio notes files as a placeholder and complete list of all possible notes.